holoviews/tests/core/testdynamic.py

```diff
@@ -1,8 +1,10 @@
 import uuid
 import time
+import sys
 from collections import deque
 
 import param
+from unittest import SkipTest
```

Comment:
Don't want to be too much of a stickler but imports are generally organized by stdlib then external libraries.

Reply:
Fixed in f9370a69c